src/pkg/notifier/handler/notification/slack_handler.go

```diff
@@ -108,7 +110,7 @@ func (s *SlackHandler) process(event *model.HookEvent) error {
 		"address":          event.Target.Address,
 		"skip_cert_verify": event.Target.SkipCertVerify,
 	}
-	return notification.HookManager.StartHook(event, j)
+	return notification.HookManager.StartHook(orm.Context(), event, j)
```

Comment:
Seems that the `context` can be passed by the caller?

Reply:
good catch, updated.